apps/charterafrica/src/components/GrantsAndFellowshipsGrid/GrantsAndFellowshipsGrid.test.js

```diff
@@ -0,0 +1,87 @@
+import { createRender } from "@commons-ui/testing-library";
+import React from "react";
+
+import GrantsAndFellowshipsGrid from "./GrantsAndFellowshipsGrid";
+
+import theme from "@/charterafrica/theme";
+
+// eslint-disable-next-line testing-library/render-result-naming-convention
+const render = createRender({ theme });
+
+const defaultProps = {
+  title: "Fellowships",
+  grants: [
```

Comment:
Why would title be **Fellowships** and items be `grant`? I'm expecting either they'll also be named `fellowships` or if we want a generic name applicable to everything (grants, fellowships and events), `items` is usually a good start.

Reply:
I agree. Items is a good one.